packages/amplify-category-function/package.json

```diff
@@ -18,7 +18,9 @@
     "amplify-category-storage": "2.2.1",
     "fs-extra": "^8.1.0",
     "graphql-transformer-core": "6.14.1",
+    "cron-builder": "^0.3.0",
```

Comment:
This is a 3 years old package, can we find a better actively maintained version?